public/modules/interactions/brancheFiltresMesures.mjs

```diff
@@ -1,7 +1,11 @@
 const CLASSE_ELEMENT_INVISIBLE = 'invisible';
 
-const mesureSpecifiqueDeCategorie = (elementMesureSpecifique, categorieFiltre) => !categorieFiltre
-  || $(`option[value="${categorieFiltre}"]:selected, option[value=""]:selected`, elementMesureSpecifique).length === 1;
+const mesureSpecifiqueDeCategorie = (elementMesureSpecifique, categorieFiltre) => {
+  const $categories = $(":radio[name^='categorie-']", elementMesureSpecifique);
```

Comment:
La doc jQuery dit :

> As with other pseudo-class selectors (those that begin with a `:`) it is recommended to precede it with a tag name or some other selector; otherwise, the universal selector (`*`) is implied. In other words, the bare `$( ":radio" )` is equivalent to `$( "*:radio" )`, so `$( "input:radio" )` should be used instead.

Ça me rassurerait qu'on aille en ce sens et qu'on écrive `… = $("input:radio[…]", …)`. Qu'en penses-tu ?

Reply:
Oui bien vu, je fais la modif 👍🏼